site/pages/create-disruption.tsx

```diff
@@ -1,5 +1,211 @@
+                                    className="govuk-radios__input"
+                                    id="disruption-planned"
+                                    name="disruptionType"
+                                    type="radio"
+                                    value="Planned"
+                                />
+                                <label className="govuk-label govuk-radios__label" htmlFor="disruption-planned">
+                                    Planned
+                                </label>
+                            </div>
+                            <div className="govuk-radios__item">
+                                <input
+                                    className="govuk-radios__input"
+                                    id="disruption-unplanned"
+                                    name="disruptionType"
+                                    type="radio"
+                                    value="Unplanned"
+                                />
+                                <label className="govuk-label govuk-radios__label" htmlFor="disruption-unplanned">
+                                    Unplanned
+                                </label>
+                            </div>
+                        </div>
+                        <div className="govuk-form-group">
+                            <label className="govuk-label" htmlFor="summary">
+                                Summary
+                            </label>
+                            <input className="govuk-input w-3/4" id="summary" name="summary" type="text" />
+                        </div>
+                        <div className="govuk-form-group">
+                            <label className="govuk-label" htmlFor="description">
+                                Description
+                            </label>
+                            <textarea className="govuk-textarea w-3/4" id="description" name="description" rows={5} />
+                        </div>
+                        <div className="govuk-form-group">
+                            <label className="govuk-label" htmlFor="associated-link">
+                                Associated Link
+                            </label>
+                            <input
+                                className="govuk-input w-3/4"
+                                id="associated-link"
+                                name="associatedLink"
+                                type="text"
+                            />
+                        </div>
+                        <div className="govuk-form-group">
+                            <label className="govuk-label" htmlFor="summary">
+                                Reason for disruption
+                            </label>
+                            <select className="govuk-select w-3/4" id="disruption-reason" name="disruptionReason">
+                                {Object.keys(Reason).sort().map(renderOption)}
```

Comment:
will the select options not need a value also? i.e. the value would be the key of the enum

Reply:
Value added for options